core/primitives/block_id.hpp

```diff
@@ -13,7 +13,7 @@
 
 namespace kagome::primitives {
   /// Block id is the variant over BlockHash and BlockNumber
-  using BlockId = std::variant<common::Hash256, BlockNumber>;
+  using BlockId = boost::variant<common::Hash256, BlockNumber>;
```

Comment:
As we moved to boost variant remove include variant above

Reply:
done